src/interfaces/IMetaMorpho.sol

```diff
@@ -31,8 +31,6 @@ struct MarketAllocation {
     MarketParams marketParams;
     /// @notice The amount of assets to allocate.
     uint256 assets;
-    /// @notice The amount of shares to allocate.
-    uint256 shares;
```

Comment:
We should also change the dev comment above the  struct @MerlinEgalite

Reply:
cc @Rubilmax